Bottom: 7b527f711768166a0eaa950c02710df6fdd564f1
Top:    5ba672da6312ca4105dd8000b9555554974dc05e
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-20 11:54:00 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/codecs/tas5504.c b/sound/soc/codecs/tas5504.c
index d2dbdaa..6ac3289 100644
--- a/sound/soc/codecs/tas5504.c
+++ b/sound/soc/codecs/tas5504.c
@@ -820,11 +820,9 @@ static int tas5504_i2c_probe(struct i2c_client *client,
 	INIT_LIST_HEAD(&priv->codec.dapm_widgets);
 	INIT_LIST_HEAD(&priv->codec.dapm_paths);
 
-#if defined(CONFIG_SND_SOC_OF_SIMPLE)
 	/* Tell the of_soc helper about this codec */
-	of_snd_soc_register_codec(&tas5504_soc_codec_dev, priv, &tas5504_dai, 1,
+	of_snd_soc_register_codec(&tas5504_soc_codec_dev, priv, &tas5504_dai,
 				  client->dev.archdata.of_node);
-#endif
 
 	dev_dbg(&client->dev, "I2C device initialized\n");
 	return 0;
